.ocm/branch-info.yaml

```
@@ -0,0 +1,5 @@
+release-branch-template: release-v$major.$minor # e.g. release-v1.0
+branch-policy:
+  significant-part: minor # major, minor, patch
+  supported-versions-count: 2
```

```suggestion
  supported-versions-count: 1
```
I can't find any patch release we cut for the ingress-default-backend. Thus, I assume we just support the latest version.